Application icon error.

### Steps to reproduce

Changing the app icon causes an error.

### Expected behavior

It shouldn't have been.

### Actual behavior

java.lang.IllegalArgumentException: Component class com.github.libretube.activities.MainActivity does not exist in com.github.libretube
	at android.os.Parcel.createExceptionOrNull(Parcel.java:2441)
	at android.os.Parcel.createException(Parcel.java:2421)
	at android.os.Parcel.readException(Parcel.java:2404)
	at android.os.Parcel.readException(Parcel.java:2346)
	at android.content.pm.IPackageManager$Stub$Proxy.setComponentEnabledSetting(IPackageManager.java:7766)
	at android.app.ApplicationPackageManager.setComponentEnabledSetting(ApplicationPackageManager.java:3267)
	at com.github.libretube.ui.preferences.AppearanceSettings$$ExternalSyntheticLambda2.onPreferenceChange(R8$$SyntheticClass:11)
	at androidx.preference.Preference.callChangeListener(Unknown Source:4)
	at com.github.libretube.ui.views.MaterialPreferenceFragment$$ExternalSyntheticLambda0.onClick(R8$$SyntheticClass:6)
	at androidx.appcompat.app.AlertController$AlertParams$3.onItemClick(AlertController.java:1)
	at android.widget.AdapterView.performItemClick(AdapterView.java:376)
	at android.widget.AbsListView.performItemClick(AbsListView.java:1282)
	at android.widget.AbsListView$PerformClick.run(AbsListView.java:3528)
	at android.widget.AbsListView$3.run(AbsListView.java:4691)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8669)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
Caused by: android.os.RemoteException: Remote stack trace:
	at com.android.server.pm.PackageManagerService.setEnabledSetting(PackageManagerService.java:28994)
	at com.android.server.pm.PackageManagerService.setComponentEnabledSetting(PackageManagerService.java:28731)
	at android.content.pm.IPackageManager$Stub.onTransact(IPackageManager.java:3545)
	at com.android.server.pm.PackageManagerService.onTransact(PackageManagerService.java:9919)
	at android.os.Binder.execTransactInternal(Binder.java:1215)

    

### LibreTube version

0.6.0

### Android version

Android 12

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

--- a/app/src/main/java/com/github/libretube/util/ThemeHelper.kt
+++ b/app/src/main/java/com/github/libretube/util/ThemeHelper.kt
@@ -100,7 +100,7 @@ object ThemeHelper {
         for (activityAlias in activityAliases) {
             val activityClass = "com.github.libretube." +
                 if (activityAlias == activityAliases[0]) {
-                    "activities.MainActivity" // default icon/activity
+                    "ui.activities.MainActivity" // default icon/activity
                 } else {
                     activityAlias
                 }
@@ -116,7 +116,7 @@ object ThemeHelper {
         // set the class name for the activity alias
         val newLogoActivityClass = "com.github.libretube." +
             if (newLogoActivityAlias == activityAliases[0]) {
-                "activities.MainActivity" // default icon/activity
+                "ui.activities.MainActivity" // default icon/activity
             } else {
                 newLogoActivityAlias
             }

